docs/keto/engines/acl.md

```diff
@@ -9,16 +9,16 @@ is a matrix of users and permissions:
 
 |       | blog_post.create | blog_post.delete | blog_post.modify | blog_post.read |
 | ----- | ---------------- | ---------------- | ---------------- | -------------- |
-| alice | yes              | yes              | yes              | yes            |
-| bob   | no               | no               | no               | yes            |
-| peter | yes              | no               | yes              | yes            |
+| Alice | yes              | yes              | yes              | yes            |
```

Comment:
One liberty I took is capitalizing people's names, at least in text. In code, they could still be lowercase since it's data. But in Markdown it reads really weird if they are lowercase.